lib/request_handler/fieldsets_parser.rb

```diff
@@ -29,13 +29,18 @@ def run
     private
 
     def parse_options(type, values)
+      raise ExternalArgumentError, fieldsets: "invalid type: <#{type}>" if allowed[type] == false
```

Comment:
I think for the client it should not make a difference if we define the fieldsets like this

```
fieldsets do
  allowed do
    sample true
    run_session false
  end
end
```
or like this

```
fieldsets do
  allowed do
    sample true
  end
end
```

Usually one should prefer the second version, but I felt like if we allow this being set to `true` with the meaning that all fields are allowed we also need to define what `false` means.